experimental/overhead_matching/kimera_vio_pipeline_test.cc

```diff
@@ -7,17 +7,19 @@
 #include <utility>
 
 #include "experimental/overhead_matching/kimera_spectacular_data_provider.hh"
+#include "fmt/core.h"
 #include "kimera-vio/dataprovider/DataProviderInterface.h"
 #include "kimera-vio/logging/Logger.h"
 #include "kimera-vio/pipeline/Pipeline.h"
 #include "kimera-vio/pipeline/RgbdImuPipeline.h"
 #include "kimera-vio/utils/Statistics.h"
 #include "kimera-vio/utils/Timer.h"
 
-DEFINE_string(params_folder_path, "../params/Euroc",
+DEFINE_string(params_folder_path, "/home/ekf/software/robot/data/iphoneSpectacularParams",
               "Path to the folder containing the yaml files with the VIO parameters.");
-DEFINE_string(dataset_path, "/Users/Luca/data/MH_01_easy",
-              "Path of dataset (i.e. Euroc, /Users/Luca/data/MH_01_easy).");
+DEFINE_string(dataset_path, "/home/ekf/software/robot/data/Walk-to-work",
+              // DEFINE_string(dataset_path, "/home/ekf/software/robot/data/20241212_090634",
+              "Path of dataset");
```

Comment:
I've been using the cxxopts library to handle argument flags. Up to you if you want to switch to that. Here is an example: https://github.com/ewfuentes/robot/blob/43f218083740bdac2fd7ab53ec4dd6b8362e422a/experimental/pokerbots/pokerbot_master.cc#L213

I see that this isn't a `cc_test`. Is there something that is making it difficult to do that instead? I could imagine it being useful to have a binary to run Kimera on some log.

Reply:
Unfortunately cxxopts and google flags don't play well together, we would likely have to add a bandaid or remove all gflags from used kimera components. 

Re `cc_test` vs `cc_binary` -- I picked cc_binary because I thought it would be useful to have a binary to run on logs. If we change it to a cc_test would we still be able to use it to get vio solutions from different bags?

I can make it a test and make sure the final pose is somewhat reasonable. Where should I put the iphone spectacular params? (this repo, pulled in as a zip, standalone repo?)

Reply:
in another pr break out an interface to create and run pipeline on a spec log. Create a test that uses this, and use this interface in the binary as well.